serverlessworkflow/modules/ROOT/pages/core/defining-an-input-schema-for-workflows.adoc

```diff
@@ -1 +1,58 @@
-//= Defining an input schema for workflows
\ No newline at end of file
+Defining an input schema for workflow
+=====================================
+:compat-mode!:
+
+DataInputSchema is one of the parameters in the Workflow definition as per the Serverless Workflow specification.
+
+The dataInputSchema property can be used to validate the workflow data input against a defined JSON Schema. dataInputSchema can be defined as follows:
+[source,json]
+----
+"dataInputSchema": {
+   "schema": "URL_to_json_schema",
+   "failOnValidationErrors": false
+}
+----
+
+"schema" property is an URI, which holds the path to the JSON schema used to validate the workflow data input. The JSON schema file should be placed in the resources section of the project.
+
+An example for a workflow definition with datainputschema can be found here- https://github.com/kiegroup/kogito-runtimes/blob/main/quarkus/extensions/kogito-quarkus-serverless-workflow-extension/kogito-quarkus-serverless-workflow-integration-test/src/main/resources/expression.sw.json#L16[GitHub repository]
```

Comment:
I think we have an attribute to link our examples, see: 
https://raw.githubusercontent.com/kiegroup/kogito-docs/main/serverlessworkflow/modules/ROOT/pages/core/understanding-jq-expressions.adoc

Example:
```asciidoc
link:{kogito_sw_examples_url}/serverless-workflow-greeting-quarkus[`serverless-workflow-greeting-quarkus`] 
```

Reply:
Would be better to link to an example rather than our main repo.